Postgresql+postgresql-admin

fernandosanchezmunoz opened the pull request:
added:
- separate sections for "persistence" and "external_access" for clarity
- visible comments on unencrypted external connections
- removed VHost for postgres (which is TCP) -- not for the admin interface (HTTP)
- Improved documentation

added:
- minDcosVersion: 1.7
- packagingVersion: 3.0
- Persistent storage (on postgres) AND "external_access" (on both, for automatic VIP on MLB) are both optional and default to FALSE. Added disclaimer about unencrypted connection.


Based on:
postgres:9.6
zhajor/docker-phppgadmin

pyronicide commented on repo/packages/P/postgresql/0/marathon.json.mustache:
```diff
@@ -0,0 +1,63 @@
+  "labels": {
+    "DCOS_SERVICE_NAME": "{{service.name}}",
+    {{#networking.external_access}}
+    "HAPROXY_0_VHOST": "{{networking.virtual_host}}",
```
This only works for HTTP. You'll need to use service ports.

fernandosanchezmunoz replied:
removed VHost from postgres (not from postgres-admin).

fernandosanchezmunoz commented:
added:

separate sections for "persistence" and "external_access" for clarity
visible comments on unencrypted external connections
removed VHost for postgres (which is TCP) -- not for the admin interface (HTTP)
Improved documentation
